src/bandwidthRtc.ts

```diff
@@ -91,7 +101,10 @@ class BandwidthRtc {
     const peerConnection = new RTCPeerConnection(RTC_CONFIGURATION);
     this.setupNewPeerConnection(peerConnection, endpointId, mediaTypes);
     mediaStream.getTracks().forEach((track) => {
-      peerConnection.addTrack(track, mediaStream);
+      var sender = peerConnection.addTrack(track, mediaStream);
+      if (track.kind === "audio") {
+        this.dtmfSender = new DTMFSender(sender);
```

Comment:
My vote is for camelCaseConsistency so:
```this.dtmfSender = new DtmfSender(sender);```

Reply:
I explicitly git renamed the file (instead of deleting it and adding a new file with a camel-cased name) but GitHub (or my Windows git client) messed up and added a new file 🤷‍♂️ 🤦‍♂️